lib/triggers/notificationTrigger.js

```diff
@@ -258,23 +254,22 @@ NotificationTrigger.prototype._respond = function(toId,userId,message) {
 					text: "Please send me the following command to validate "+that.db[userId].email+" as the notification email for "+that._username(userId)+"/"+userId+"\r\n\r\n"+that.options.cmd+" emailconfirm "+that.hashfunc(email)+"\r\n\r\n\r\nPlease do not reply to thie email"
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Line is too long.](https://www.codacy.com/app/node-steam-chat-bot/steam-chat-bot/file/946709259/issues/source?bid=1980036&fileBranchId=2848073#l254)